README.md

```diff
@@ -77,21 +80,26 @@ Look at the Moodle docs regarding the protocol that is being used for a more det
 Rest example:
 Must use the POST protocol.
 
-```
+```sh
 curl -d "wstoken=value1&wsfunction=mod_applaunch_complete_activity&moodlewsrestformat=json&activityslug=value2" -X POST https://www.moodle.com/webservice/rest/server.php
 ```
 Calling this endpoint with the external function `archivecompletion` will mark the activity as complete for the user who clicked the launch button.
 
 On a success you will receieve (if JSON is selected for return format)
 
-```
+```yaml
```

Comment:
technically json? even though its a super set, dunno if github renders them any different

Reply:
So, it needed to be either js or yaml for syntax highlighting to kick in, otherwise it would stay broken or look like just text.

Reply:
The reason why I opted away from using `js` is because it's not really "javascript" code. It's just data for it. 

Using `yaml` as the key seemed like an easy way to enable highlighting and even if you forget the quotes or trailing commas, it will at least keep the highlighting to get the main point across. JS _might_ not do that.